### DISCUSSION TITLE
how to implement a webkit in styledcomponents?

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#85] [Web] [answered]
Guten Abend zusammen, 

ich habe folgendes Problem (siehe Bild) dass ich einen Fehler angezeigt bekommen, dass ich dieses tool nicht benutzen sollte, es aber trotzdem ausführt was ich möchte. Weiss jemand wie ich dieses übergehen kann? 
Vielen Dank im Voraus! 
![Screenshot 2022-08-18 at 21 16 19](https://user-images.githubusercontent.com/102238180/185476603-7a7a48db-ea07-4c36-9993-bcd0f80e1c3b.png)

**Comment by fabian-hanso** [comment, marked as answer]
Habe dir dazu mal zwei Seiten rausgesucht. Hier wird in der ersten Warnung darauf hingewiesen, dass es nicht mehr dem Standard entspricht:

https://developer.mozilla.org/de/docs/Web/CSS/appearance

und hier sieht man, wie es damals angewendet wurde. Heute gibt es andere Herangehensweisen, die für jeden User in den jeweiligen Browsern supported werden.

https://developer.mozilla.org/en-US/docs/Web/CSS/appearance

**Reply by PhilipZi** [reply]
vielen Dank nochmal für die Antwort aber das dieses feature nicht wirklich verwendet werden soll habe ich ja schon aus der Meldung entnommen, aber eine Lösung zu meinem Problem konnte ich jetzt nicht auf den Webseiten finden. 
""Heute gibt es andere Herangehensweisen, die für jeden User in den jeweiligen Browsern supported werden."" Und wo werden diese dokumentiert? Ich habe sehr viel researches unternommen, aber leider bin ich auf keine Alternative gekommen, es wurde überall mit dem Webkit dokumentiert oder eben nicht mit Styledcomponents zu arbeiten. Wäre echt schade, wenn es keine andere Lösung Variante gibt.

**Reply by PhilipZi** [reply]
Ich muss mich entschuldigen, ich habe zu kompliziert gedacht gestern Abend und habe es heute morgen bei erneutem anschauen lösen können dank der zweiten Doku! Vielen Dank!!!!

**Comment by F-Kirchhoff** [comment]
Ein kleiner Hinweis: um in styled components das element selbst zu stylen brauchst du nicht den `&` Selector zu verwenden. schreib das CSS einfach nicht verschachtelt da hin:
```css
-webkit-appearance: none;
...
&:focus {
  ...
}
```
